Pipfile

```diff
@@ -9,8 +9,7 @@ verify_ssl = true
 name = "test-pypi"
 
 [dev-packages]
-aiohttp = "==3.6.2"
-aries-cloudagent = "==0.4.5"
```

Comment:
unfortunately, `aries-cloudagent` [requires Markdown~=3.1.1](https://github.com/hyperledger/aries-cloudagent-python/blob/master/requirements.txt#L7), whereas we have `Markdown>=3.2.1`